### ISSUE TITLE
document services running on sudomesh server to prepare for redeployment

### ISSUE THREAD

**Issue by paidforby** [sudomesh/bugs#39]
Based on discussion from #28. The psychz exit node will soon be deprecated and we could use another exit node that is already in most home nodes' `/etc/config/tunneldigger`. builds.sudomesh.org fits this description and is the most used and accessible to current sudomeshers.  

With that in mind, we need to document the services that are running on builds.sudomesh.org that need to be re-deployed on a new, hopefully, less expensive server.

Here's a short list of what I can think of:
* People's Open Website: https://peoplesopen.net
* BYOI website: https://buildyourowninter.net
* disater.radio website: https://disaster.radio
* Etherpad-lite instance: https://peoplesopen.net/pad
* Rocket Chat instance: https://peoplesopen.net/chat
* sudowrt-firmware builds storage: https://builds.sudomesh.org
* meshygardentoolshed, https://peoplesopen.net/gardenmesh/chart/

Will add comments for links to documentation for each of these.
Please add more things if you think of them.

**Comment by bennlich** [comment]
Is the https://sudomesh.org wiki also hosted on this machine?

**Comment by jnny** [comment]
@bennlich nope, it's hosted on sudo room's wiki, which has its own DO droplet.

**Comment by paidforby** [comment]
For reference, the server hosting these websites and services have successfully been redirected to 107.170.221.27 so we can now use the old IP address (107.170.219.5) as a new backup exit node.
